File: exercises/beecrowd/1010/1010.ts
// https://judge.beecrowd.com/pt/problems/view/1010
import * as fs from 'fs';

const input = fs.readFileSync('/dev/stdin', 'utf8');
const lines = input.split(/\r?\n/);

const line1 = String(lines.shift()).split(' ');
const line2 = String(lines.shift()).split(' ');

const total = (Number(line1[1]) * Number(line1[2])) + (Number(line2[1]) * Number(line2[2]))

console.log("VALOR A PAGAR: R$", total.toFixed(2))

File: exercises/beecrowd/1010/1010.go
// https://judge.beecrowd.com/pt/problems/view/1010
package main

import "fmt"

func main() {
	var id1, id2, quantity1, quantity2 int
	var value1, value2 float64

	fmt.Scanf("%d %d %f ", &id1, &quantity1, &value1)
	fmt.Scanf("%d %d %f", &id2, &quantity2, &value2)

	total := (float64(quantity1) * value1) + (float64(quantity2) * value2)

	fmt.Printf("VALOR A PAGAR: R$ %.2f\n", total)
}
